managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -2753,7 +2753,8 @@ synchronized List<Long> internalEvictOffloadedLedgers() {
                     ReadHandle readHandle = ledger.join();
                     if (readHandle instanceof OffloadedLedgerHandle offloadedLedgerHandle) {
```

Comment:
`BlobStoreBackedReadHandleImpl` is the only production code class that currently implements `OffloadedLedgerHandle`.